ui_framework/components/local_nav/_local_nav.scss

```diff
@@ -16,6 +16,11 @@
     color: $localNavTextColor--darkTheme;
     background-color: $localNavBackgroundColor--darkTheme;
   }
+
+  h1.kuiLocalBreadcrumb {
+    font-size: 14px;
+    margin: 0;
+  }
```

Comment:
Could you move these classes to the `.kuiLocalBreadcrumb` class so that we don't need to couple them so tightly to the `h1` element?

You'll also need to run `npm run uiFramework:start` in terminal tab, which will generate the CSS from this UI Framework. Could you also commit that generated CSS please?